data-model/SDP-sample-records/script_sample1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/script/1",
+  "label": "Serto (script)",
+  "associated_writing_systems": ["Syriac"],
```

Comment:
controlled vocabulary. Potentially could have a script be used for multiple writing systems (e.g., Naskh to write Arabic and Persian)